### ISSUE TITLE
ListContainers fails on lack of domain

### ISSUE THREAD

**Issue by jellonek** [Mirantis/virtlet#160]
We should return empty list when requested sandbox or container is not found, instead of returning error like we are doing this right now.

**Comment by jellonek** [comment]
Looks like it's not resolved by #162 
```
libvirt: QEMU Driver error : Domain not found: no domain with matching uuid 'c867d8d4-0f6b-452f-6ef6-d30389aee38b' (c867d8d4-0f6b-452f-6ef6-d30389aee38b-cirros)
libvirt: QEMU Driver error : Domain not found: no domain with matching uuid 'c867d8d4-0f6b-452f-6ef6-d30389aee38b' (c867d8d4-0f6b-452f-6ef6-d30389aee38b-cirros)
E1216 16:05:51.071189   21079 manager.go:356] Error when listing containers with filter (*runtime.ContainerFilter)(0xc420268b00)({
 Id: (*string)(<nil>),
 State: (*runtime.ContainerState)(<nil>),
 PodSandboxId: (*string)(<nil>),
 LabelSelector: (map[string]string) (len=1) {
  (string) (len=21) "io.kubernetes.managed": (string) (len=4) "true"
 },
 XXX_unrecognized: ([]uint8) <nil>
})
: Domain not found: no domain with matching uuid 'c867d8d4-0f6b-452f-6ef6-d30389aee38b' (c867d8d4-0f6b-452f-6ef6-d30389aee38b-cirros)
E1216 16:05:51.071188   21079 manager.go:356] Error when listing containers with filter (*runtime.ContainerFilter)(0xc420268680)({
 Id: (*string)(<nil>),
 State: (*runtime.ContainerState)(0xc42042a9c0)(1),
 PodSandboxId: (*string)(<nil>),
 LabelSelector: (map[string]string) (len=1) {
  (string) (len=21) "io.kubernetes.managed": (string) (len=4) "true"
 },
 XXX_unrecognized: ([]uint8) <nil>
})
: Domain not found: no domain with matching uuid 'c867d8d4-0f6b-452f-6ef6-d30389aee38b' (c867d8d4-0f6b-452f-6ef6-d30389aee38b-cirros)
```
Note: there are two calls with filter not empty, once with status empty (ok), once with status set to RUNNING - we are not distinguishing that.

**Comment by jellonek** [comment]
Not reproducible now, closing.
